src/main/python/janusgraph_python/core/datatypes/Circle.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2018 JanusGraph Python Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+class Circle(object):
+    """
+    The Class representation of JanusGraph GeoShape Circle.
+    """
+    def __init__(self, latitude, longitude, radiusInKM):
+        """
+            Creates a GeoShape Circle with given longitude, latitude and radius in KMs.
+
+        Args:
+            latitude (float): The latitude of circle's center.
+            longitude (float): The longitude of circle's center
+            radiusInKM (int): The radius of circle in KM.
+        """
+
+        self.__latitude = float(latitude) * 1.0
```

Comment:
Can be removed, the multiplier by 1.0. That is done, just a extra layer of safety, in case the type casting didn't go through, and manually changing to float using the multiplier of 1.0.